packages/libs/src/sdk/services/Solana/programs/BaseSolanaProgram.ts

```diff
@@ -23,7 +23,7 @@ const isPublicKeyArray = (arr: any[]): arr is PublicKey[] =>
  */
 export class BaseSolanaProgram {
   /** The endpoint for the Solana RPC. */
-  protected readonly connection: Connection
+  public readonly connection: Connection
```

Comment:
Just curious why this was moved to public? Seems like this is a base class inherited by other programs, so are we trying to make it possible to grab a connection from any of the programs you are using?

Reply:
Might just be a leftover from debugging, but I'm not sure what I want to do here. I'm not in love with the inheritance pattern or having separate instances of the connection

Reply:
Ah yes it's needed so that we can call `sendTransaction` on the wallet adapter and provide the connection for the non-app flow